src/main/java/org/assertj/jodatime/api/DateTimeAssert.java

```diff
@@ -51,6 +59,125 @@ protected DateTime getActual() {
     return actual;
   }
 
+  /**
+   * Verifies that the year of the actual {@code DateTime} is equal to the given year
+   * <p>
+   * Example :
+   * <pre><code class='java'> assertThat(new DateTime(&quot;2000-01-01&quot;)).hasYear(1);</code></pre>
+   *
+   * @param other the given year.
+   * @return this assertion object.
+   * @throws AssertionError if the actual {@code DateTime} is {@code null}.
+   * @throws AssertionError if the year of the actual {@code DateTime} is not equal to the given year.
+   */
+  public DateTimeAssert hasYear(int other){
```

Comment:
did not think about it at my first review but can we name `other` parameters according to their meaning, here it should be `year` or `expectedYear` (pick the one you like more).